server/lib/host-metrics.js

```diff
@@ -414,9 +414,9 @@ export async function getPendingHostFeeShare(host, { startDate, endDate, collect
           AND t."kind" = 'HOST_FEE_SHARE_DEBT'
           AND t."deletedAt" IS NULL
           AND ts."deletedAt" IS NULL
-          AND ts."status" IN ('OWED', 'INVOICED')
-          AND t."createdAt" >= :startDate
-          AND t."createdAt" <= :endDate
+          AND ts."status" IN ('OWED')
```

Comment:
Not sure about this one. If it's `INVOICED`, then it's still pending no?

Reply:
It depend the definition. In the context of the settlement process, we don't want to Invoice multiple time.

I think it's fine to move to this definition.

Reply:
Alternative is to pass an explicit `kind` when we want it.

Reply:
I agree for the settlement script. For host metrics however, when I look at my pending host fee share, I expect to see everything that I still haven't paid - invoiced or not.

Reply:
You might be right theoretically.

However, as far as I know, we're not surfacing this metric anymore, it's only used in the context of the settlement.

Still, I took the feedback in consideration and refactored, so there is no surprise.